src/wizard/node/profiling-onboarding/node/2.configure-performance.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Node
+doc_link: https://discord.gg/zrMjKA4Vnz
+support_level: alpha
+type: language
+---
+
+#### Configure Performance
+
+Profiling depends on Sentry’s performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK:
+
+```javascript
+import * as Sentry from "@sentry/node";
+import "@sentry/tracing";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  // Set tracesSampleRate to 1.0 to capture 100%
+  // of transactions for performance monitoring.
+  // We recommend adjusting this value in production
```

Comment:
Should we have this comment for all the platforms?

Reply:
Wouldn't hurt, I can add it in.